lms/templates/courseware/courseware.html

```diff
@@ -271,3 +271,21 @@
       % endif
     </nav>
 % endif
+
+<%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
+  var welcomeLink = $("#welcome");
+  var accessDeniedUpsellLink = $("#accessDeniedUpsell");
+
+  TrackECommerceEvents.trackUpsellClick(welcomeLink, 'in_course_welcome', {
+      pageName: "in_course",
+      linkType: "link",
+      linkCategory: "welcome"
+  });
+
+  TrackECommerceEvents.trackUpsellClick(accessDeniedUpsellLink, 'in_course_upgrade', {
+    pageName: "in_course",
+    linkType: "link",
+    linkCategory: "(none)"
```

Comment:
what is none here supposed to mean?

Reply:
Same comment as on the other PR, maybe you could use the ID for this for consistency - unless none is what you actually want in this case :)

Reply:
Sorry about that, guys- I'm adding a summary doc in the comments now to give a little background. Not all links have a category, so for those oddballs we're leaving it as "(none)" for consistency

Reply:
Could you just remove the linkCategory from the hash instead?